dietpi/dietpi-software

```diff
@@ -11174,6 +11213,54 @@ _EOF_
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# User
+			local usercmd='useradd -rMU'
+			getent passwd ffsync &> /dev/null && usercmd='usermod -a'
+			$usercmd -G dietpi -d /opt/ffsync -s $(command -v nologin) ffsync
+
+			# Create database directory
+			mkdir -p $G_FP_DIETPI_USERDATA/firefox-sync-db
+
+			# Create database if it doesn't exist
+			if [[ ! -a $G_FP_DIETPI_USERDATA/firefox-sync-db/FF-Sync-DB.db ]]; then
+				sqlite3 $G_FP_DIETPI_USERDATA/firefox-sync-db/FF-Sync-DB.db "create table aTable(field1 int); drop table aTable;"
+			fi
+
+			# Modify config file
+			ffsync_ip=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')
+			dietpiUsrDataEsc=$(sed 's|/|\\/|g' <<< $G_FP_DIETPI_USERDATA)
+			sed -i.bak "s/host = 0.0.0.0/host = $ffsync_ip/" /etc/firefox-sync/syncserver.ini
+			#sed -i.bak "s/port = 5000/port = 5000/" /etc/firefox-sync/syncserver.ini
+			sed -i.bak "s/localhost:5000\//$ffsync_ip:5000\//" /etc/firefox-sync/syncserver.ini
+			sed -i.bak "s/#sqluri = sqlite:\/\/\/\/tmp\/syncserver.db/sqluri = sqlite:\/\/\/$dietpiUsrDataEsc\/firefox-sync-db\/FF-Sync-DB.db/" /etc/firefox-sync/syncserver.ini
+
+			cat << _EOF_ > /etc/systemd/system/firefox-sync.service
+[Unit]
+Description=Firefox Sync Server (DietPi)
+After=dietpi-boot.service network.target
```

Comment:
Since Stretch I would use `network-online.target` instead of `network.target`:
```
Wants=network-online.target
After=network-online.target dietpi-boot.service
```